### PR TITLE
Reveal the useful Homestead `flip` command

### PR DESCRIPTION
**Pull request by MladenJanjetovic**

### CODE REVIEW

**Comment by taylorotwell** [comment]
It's unclear how this should be used.

**Comment by MladenJanjetovic** [comment]
ok, let me rephrase it a bit

**Comment by browner12** [comment]
didn't realize this wasn't documented after I wrote it.

agree, we need to get something in there.

> Homestead comes with both Apache and Nginx already installed. Since you can only have one webserver running at a time, use the `flip` command to switch back and forth between them.

**Comment by MladenJanjetovic** [comment]
I can't edit it until it's reopened I guess. Or you should create your PR? 
Either way, this is very useful for multiple sites. Thanks

**Comment by browner12** [comment]
since Taylor closed this one, you can create another PR with your updated phrasing.

**Comment by MladenJanjetovic** [comment]
This is implementer in https://github.com/laravel/docs/pull/3979
